composer.json

```diff
@@ -26,7 +26,7 @@
         }
     ],
     "require": {
-        "php": "^7.2.5",
+        "php": "^7.2.22",
```

Comment:
This should be `~7.2.22|^7.3.9`, since PHP 7.3.0-7.3.8 is also affected by the same bug as 7.2.0-7.2.11.

Reply:
You're totally right. The constraint was updated.
Thank you.